modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/ScrollPaneSkinTest.java

```diff
@@ -860,4 +870,45 @@ public MouseEvent generateMouseEvent(EventType<MouseEvent> type,
             return event;
         }
     }
+
+    /**
+     * checks that there are no memory leaks when replacing scroll pane content
+     */
+    @Test
+    public void checkMemoryLeaks_JDK_8293444() {
```

Comment:
may be rename to something similar as : `testScrollPaneObjLeakWhenUsedSameContent`